InputManager.gd

```diff
@@ -69,37 +70,42 @@ func _unhandled_input(event):
 				var rel1 = event.position - last_mouse_press.position
 				var rel2 = rel1 + event.relative
 				emit("twist", InputEventScreenTwist.new({"position": last_mouse_press.position,
-													     "relative": rel1.angle_to(rel2),
+														 "relative": rel1.angle_to(rel2),
 														 "speed": event.speed}))
 	
 	# Touch.
 	elif event is InputEventScreenTouch:
 		if event.pressed:
 			touches[event.get_index()] = event 
 			if (event.get_index() == 0): # First and only touch.
-				emit("single_touch", InputEventSingleScreenTouch.new(event))
+				single_touch_cancelled = false
+				emit("single_touch", InputEventSingleScreenTouch.new(event, false))
 				only_touch = event
-				if tap_delay_timer.is_stopped(): tap_delay_timer.start(TAP_TIME_THRESHOLD)
```

Comment:
Please restore the one line if

Reply:
It is a common practise for any langage, for lisibility, to avoid such one lines writting.
It catches my eyes too much to let it like this ...